### PR TITLE
fix docs

### LINKED ISSUE #1439: Missing docstring error in dev docs
Hi,

It seems like docs for `RNN`, `LSTM` and `GRU` don't show up in the dev docs here: https://fluxml.ai/Flux.jl/dev/models/layers/#Recurrent-Layers-1 . It should probably be easy to fix.

Attached is a screenshot: 
Screen Shot 2020-12-26 at 5 53 30 PM

### PR DESCRIPTION
**Pull request by CarloLucibello**
fix #1439

### CODE REVIEW

**Comment by gxyd** [comment]
+1. Looks good to me.

File: docs/src/utilities.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Flux initializes convolutional layers and recurrent cells with `glorot_uniform`
 by default.
 To change the default on an applicable layer, pass the desired function with the
 `init` keyword. For example:
+
 ```jldoctest; setup = :(using Flux)
```
Comment (gxyd) [inline docs/src/utilities.md:29]: Come to think of it, would it be necessary to have `:(using Flux)` here?

**Comment by CarloLucibello** [comment]
@DhairyaLGandhi I think you have to approve before calling bors

**Review by DhairyaLGandhi — APPROVED** [review]

**Pull request merged**